src/ejabberd_config.erl

```diff
@@ -315,8 +315,14 @@ handle_table_does_not_exist_error(Table) ->
 %%--------------------------------------------------------------------
 -spec parse_file(file:name()) -> state().
 parse_file(ConfigFile) ->
-    Terms = get_plain_terms_file(ConfigFile),
-    mongoose_config_parser:parse_terms(Terms).
+    TOMLFile = filename:rootname(ConfigFile) ++ ".toml",
```

Comment:
Could the filename be already given with the `.toml` extension?

Reply:
I agree that this i a bit clunky. I just wanted it to be simple as we are still not sure if we want to support both formats or only one of them. As soon as we know that, this code will be changed.